models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+                type: list
+                elements: str
+              junos_all_new_apps:
+                description: 
+                  - Configure the application group of all newly added application signatures in the latest signature package.
+                type: bool
+              any:
+                description:
+                  - Configuring the dynamic application as any installs the policy with the application as a wildcard (default).
+                type: bool
+              none:
+                description:
+                  - Configuring the dynamic application as none ignores classification results from AppID and does not use the dynamic application in security policy lookups.
+                type: bool
+            destination_address:
+              address:
+                description:
+                  - IP address, IP address set, or address book entry, or wildcard address (represented as A.B.C.D/wildcard_mask). 
+                type: list
+                elements: str
+              any:
+                description: Any IPv4 or IPv6 address
+                type: bool
+              any_ipv4:
+                description: Any IPv4 address
+                type: bool
+              any_ipv6:
+                description: Any IPv6 address
+                type: bool
+            source_address:
+              address:
+                description:
+                  - IP address, IP address set, or address book entry, or wildcard address (represented as A.B.C.D/wildcard_mask). 
+                type: list
+                elements: str
+              any:
+                description: Any IPv4 or IPv6 address
+                type: bool
+              any_ipv4:
+                description: Any IPv4 address
+                type: bool
+              any_ipv6:
+                description: Any IPv6 address
+                type: bool
+            source_identity:
+              names:
+                description: A list of specific users and roles.
+                type: list
+                elements: str
+              any:
+                description: 
+                  - Any user or role, as well as the keywords authenticated_user, unauthenticated_user, 
+                    and unknown_user.
+                type: bool
+              authenticated_user:
+                description: All users and roles that have been authenticated.
+                type: bool
+              unauthenticated_user:
+                description: 
+                  - Any user or role that does not have an IP_address mapped to authentication sources and the authentication source is up and running.
+                type: bool
+              unknown_user:
+                description: 
+                  - Any user or role that does not have an IP address mapped to authentication sources, because the authentication source is disconnected from the SRX Series device. 
+                type: bool
+        then:
```

Comment:
then is an attribute it should have basic sub-attributes like description and type.